libs/datasets/taglib.py

```diff
@@ -293,6 +294,23 @@ def content(self) -> str:
         return json.dumps(d, separators=(",", ":"))
 
 
+@dataclass(frozen=True)
+class KnownIssueNoDate(TagInTimeseries):
+    disclaimer: str
+
+    TAG_TYPE = TagType.KNOWN_ISSUE_NO_DATE
+
+    @classmethod
+    def make_instance(cls, *, content: str) -> "TagInTimeseries":
+        content_parsed = json.loads(content)
+        return cls(disclaimer=content_parsed["disclaimer"])
+
+    @property
+    def content(self) -> str:
+        d = {"disclaimer": self.disclaimer}
```

Comment:
here too?  Actually, does this end up showing up in the API?  Do we already use "disclaimer" for `KnownIssue`?  Should we switch them both to "public_note"?  Not sure...  WDYT?

Reply:
`disclaimer` is already used in `KnownIssue`. Agreed that public_note is better. I changed KnownIssueNoDate now and added a TODO for KnownIssue because changing it seems out of scope for this PR.